src/query/pipeline/sources/src/processors/sources/input_formats/impls/input_format_parquet.rs

```diff
@@ -340,6 +346,8 @@ impl AligningStateTrait for AligningState {
         if let Some(rb) = read_batch {
             if let ReadBatch::Buffer(b) = rb {
                 self.buffers.push(b)
+            } else {
+                unreachable!()
```

Comment:
Return error?

Reply:
is `ErrorCode::Internal` suitable for this?

Reply:
Ok

Reply:
done